### PR TITLE
Release 0.20.0

### PR DESCRIPTION
**Pull request by jku**
Supersedes #1723, changes from there:
* some new merges in develop
* updated changelog with those merges
* updated README to better reflect current project status (this could have been a separate PR admittedly)
* move 1.0.0 announcement to docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,31 +19,22 @@ This repository is the **reference implementation** of
...
+    metadata and to handle (de)serialization from/to files.
+  * [`tuf.ngclient`](https://theupdateframework.readthedocs.io/en/latest/api/tuf.ngclient.html),
+    a client implementation built on top of the metadata API.
+
+High-level support for implementing
+[repository operations](https://theupdateframework.github.io/specification/latest/#repository-operations)
+is planned but not yet provided: see [1.0.0 plans](docs/1.0.0-ANNOUNCEMENT.md).
+
+In addition to these APIs the project also provides a *legacy
+implementation* with`tuf.client` implementing the client workflow and
```
Comment (lukpueh) [inline README.md]:
```suggestion
implementation* with `tuf.client` implementing the client workflow and
```

File: README.md [hunk 2]
```diff
@@ -19,31 +19,22 @@ This repository is the **reference implementation** of
 It is written in Python and intended to conform to version 1.0 of the
 [TUF specification](https://theupdateframework.github.io/specification/latest/).
```
Comment (lukpueh) [inline README.md:20]: The link to 1.0.0-ANNOUNCEMENT.md also needs to change at the top of this document.

**Review by lukpueh — CHANGES_REQUESTED** [review]

**Comment by sechkova** [comment]
Except the broken link that Lukas pointed, it looks ok to me.

**Comment by jku** [comment]
only changes are the typo and link fix lukas pointed out

**Review by lukpueh — APPROVED** [review]

**Pull request merged**